torch/_inductor/codegen/wrapper.py

```diff
@@ -260,6 +260,11 @@ def __init__(self):
                 from torch import empty_strided, as_strided, device
                 from {codecache.__name__} import AsyncCompile
 
+                # multiprocess triton crashes without setting default device
```

Comment:
hopefully this is fixed by https://github.com/pytorch/pytorch/pull/90934 and i can remove this